src/server/rest/v1/router/api/UserRouter.ts

```diff
@@ -64,7 +64,11 @@ export default class UserRouter {
 
   private buildRouteUserDefaultCarTag(): void {
     this.router.get(`/${ServerRoute.REST_USER_DEFAULT_TAG_CAR}`, async (req: Request, res: Response, next: NextFunction) => {
-      req.query.ID = req.params.id;
```

Comment:
Do not remove req.query.ID assignment as it has to work in the future when www and app will be fixed.

Reply:
To not forget to change the WWW et App to use this URL path correctly